documentation/documentation_config.ts

```diff
@@ -149,4 +149,27 @@ export const Examples: Example[] = [
     codeFiles: ['./examples/column-format/no_matchers.ts'],
     categories: [Category.ColumnFormat],
   },
+  {
+    contentFile: './examples/authentication/authentication.md',
+    codeFiles: [
+      './examples/authentication/system_authentication.ts',
+      './examples/authentication/user_authentication.ts',
+    ],
+    categories: [Category.Authentication],
```

Comment:
I forgot to mention this before, but we need to specify trigger words here too, right? I.e. the word(s) that you can hover over to be prompted to open this example in the side panel.

Reply:
Ran into this issue when connecting 😆